CvGameCoreDLL/CvCityAI.cpp

```diff
@@ -4052,6 +4052,12 @@ int CvCityAI::AI_buildingValueThreshold(BuildingTypes eBuilding, int iFocusFlags
 						{
 							iTempValue += ((kBuilding.getBonusYieldModifier(iJ, iI) * getBaseYieldRate((YieldTypes)iI)) / 12);
 						}
+
+						// 1SDAN
+						if (kBuilding.getBonusYieldChange(iJ, iI) > 0)
+						{
+							iTempValue += (kBuilding.getBonusYieldChange(iJ, iI) * countNumBonusPlots() * getBaseYieldRate((YieldTypes)iI) * 6);

```

Comment:
Why is it multiplied by the base yield rate? That's definitely not right.

Reply:
I may or may not thought that represented the value of the yield to the AI. Fix incoming.

Reply:
It's actually the total yield produced by the city. It shows up in a number of other building effect evaluations because they are a percentage change that affects the total.

Reply:
For some reason I didn't notice the top level comment before. Will do.